src/main/java/com/auth0/RequestProcessor.java

```diff
@@ -37,20 +37,27 @@
     private final String responseType;
     private final AuthAPI client;
     private final IdTokenVerifier tokenVerifier;
+    private final boolean legacySameSiteCookie;
 
     @VisibleForTesting
-    RequestProcessor(AuthAPI client, String responseType, IdTokenVerifier.Options verifyOptions, IdTokenVerifier tokenVerifier) {
+    RequestProcessor(AuthAPI client, String responseType, IdTokenVerifier.Options verifyOptions, IdTokenVerifier tokenVerifier, boolean legacySameSiteCookie) {
```

Comment:
rename param to `useLegacySameSiteCookie`

Reply:
Renamed.